code/game/objects/machinery/cryopod.dm

```diff
@@ -336,15 +336,14 @@
 	if(flags_item & (ITEM_ABSTRACT|NODROP|DELONDROP) || (is_type_in_typecache(src, GLOB.do_not_preserve_empty) && !length(contents)))
 		items -= src
 		qdel(src)
-	else
+	else if(move_loc)
 		loc = null
```

Comment:
Why doesn't it use the moveToNullSpace proc?

Reply:
I wasn't aware moveToNullSpace() is a thing now. Thanks for telling.